dev-requirements.txt

```diff
@@ -5,6 +5,6 @@ isort == 5.10.1
 memray == 1.6.0; platform_system != "Windows" and sys_platform != "darwin" and platform_machine != "arm64" and platform_machine != "aarch64"
 moto==4.1.12
 pre-commit == 2.20.0
-pytest == 7.2.0
-pytest-cov == 4.0.0
+pytest ~= 7.4.2
```

Comment:
Why are these changes?

Reply:
Rolling back - was seeing if there was a latency improvement in the testing stage of CI by bumping up the version